include/RE/Offsets/Offsets.h

```diff
@@ -24,6 +24,7 @@ namespace RE
 			inline constexpr REL::ID GetLevel(static_cast<std::uint64_t>(36344));
 			inline constexpr REL::ID HasPerk(static_cast<std::uint64_t>(36690));
 			inline constexpr REL::ID IsRunning(static_cast<std::uint64_t>(36252));
+			inline constexpr REL::ID RemoveSpell(static_cast<std::uint64_t>(37772));
```

Comment:
Please just inline the id in the call. `Offsets.h` is a relic from before I used ids for relocation.

Reply:
OK, thanks for the heads-up! I updated it.